python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -7,7 +6,8 @@
 from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList, makeGangaListByRef
 
 # config_scope is namespace used for evaluating simple objects (e.g. File)
-from Ganga.Utility.Config import config_scope
+#from Ganga.Utility.Config import config_scope
```

Comment:
Probably, this PR is still in dev, I'm examining the impact of some of the changes